invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/index.js

```diff
@@ -710,8 +736,9 @@ class CommunityProfileForm extends Component {
                     loading={isSubmitting}
                     labelPosition="left"
                     primary
-                    type="submit"
+                    type="button"
                     icon
+                    onClick={(event) => handleSubmit(event)}
```

Comment:
Needed otherwise the required fields in inputs were validated by the built-in function instead of going to the backend first i.e you couldn't customize the validation.